### DISCUSSION TITLE
Questions about Resistor Ladder Network settings.

### DISCUSSION THREAD

**Discussion by eplatero** [alf45tar/PedalinoMini#431] [Q&A]
Wiring up my controller using the ladder setting referenced in the schematic diagram using the following Resistors:
2k,3k,5.1k,10k,30k

I'm not sure what settings to change to reflect these resistors in the _Options_ tab for Pedalino in the _Resistor Ladder Network_ settings. I can't enter a number above 1023

**Comment by alf45tar** [comment]
The values in the Resistor Ladder Network settings are the values coming from the ADC. They can be calculated upfront but it is not easy to consider the resistor tolerance and the non-linear ADC curve.

I suggest to use the learn mode as detailed here https://github.com/alf45tar/PedalinoMini#booting-modes (boot mode 7).

Release the button when display shows "Release for Ladder Config" https://www.youtube.com/watch?v=DUrQXEE7L9Q

**Comment by upgrader-dev** [comment]
Hello. Can't figure out what is the "Latch emulation" option under the pedal choosen as a ladder. 
The hidden question is : can I use the ladder with latch switches?

**Comment by alf45tar** [comment]
You cannot use latch switches with resistor ladder. Only momentary switches.
Latch emulation is a software feature and can be used also with ladder: first press is a PRESS action, second press is a RELEASE action, and so on.
